docs/book/src/cronjob-tutorial/testdata/project/config/default/kustomization.yaml

```diff
@@ -94,6 +105,17 @@ replacements:
          delimiter: '.'
          index: 1
          create: true
+     - select:
+         kind: ServiceMonitor
+         group: monitoring.coreos.com
+         version: v1
+         name: controller-manager-metrics-monitor
```

Comment:
If we will add the fixed name here
Could we not add the fixed name in the PATCH since it is always the same?